rankers/BiMatchRanker/__init__.py

```diff
@@ -58,11 +57,9 @@ class BiMatchRanker(Chunk2DocRanker):
 
     .. warning:: Here we suppose that the smaller chunk score means the more similar.
     """
-    query_required_keys = ('length', )
-    match_required_keys = ('length', )
 
     def __init__(self, d_miss: Optional[Union[int, float]] = None, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super().__init__(('length', ), ('length', ), *args, **kwargs)
```

Comment:
can u pass named arguments? like this if arguments change order in the future, this will still work

Reply:
Like this?
`super().__init__(query_required_keys=('length', ), match_required_keys=('length', ), *args, **kwargs)`